src/ui-presets/playback.js

```diff
@@ -42,6 +43,14 @@ export function playbackUI(props: any): React$Element<any> {
         <UnmuteIndication player={props.player} />
         <OverlayAction player={props.player} />
         <PlaybackControls player={props.player} />
+        {props.state.engine.playlist &&
+        props.state.engine.playlist.next &&
```

Comment:
The preset become more and more unreadable. 
Create static isSupported in the component class.

Reply:
@dan-ziv done